Content.Shared/Revenant/Components/RevenantComponent.cs

```diff
@@ -204,6 +204,17 @@ public sealed partial class RevenantComponent : Component
     [DataField]
     public ProtoId<AlertPrototype> EssenceAlert = "Essence";
 
+    // Begin DeltaV Changes
+    [DataField]
+    public FixedPoint2 SleepCost = 5;
+
+    [DataField]
+    public Vector2 SleepDebuffs = new(2, 5);
```

Comment:
This being a Vector2 without any documentation is painful
Just split it into two fields